### ISSUE TITLE
Dev Branch: Long press to settings page not working

### ISSUE THREAD

**Issue by rleongcs** [Blackymas/NSPanel_HA_Blueprint#905]
@edwardtfn Can you confirm that the settings page is broken, I was not able to get to the settings page with the long press for the latest `Dev` or even the one before the latest.

**Comment by edwardtfn** [comment]
Have you updated your TFT?

**Comment by rleongcs** [comment]
Hmm...I did for both the Dev versions but during update, I looked a little different than before. Please see below logs.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/61628312/052541a0-6ddb-4a6d-b20b-6da22d1dd5e6)

I am going to test now with a blank update and then update to eu again.

**Comment by rleongcs** [comment]
Blank update is the same issue. I then power cycle the switch and now I can update properly already. Also confirmed the Settings page is working and not broken. 😆 

Something must have broke it during a previous Dev update. I have to power cycle and flash multiple times before I could get the screen up. Even though it was working fine after that, I guessed it did something and prevented the update. 

Again...looks like Power Cyling the switch is the solution.

**Comment by edwardtfn** [comment]
These TFT uploads are always challenging. 😩
But anyways, I've made a change in that call to the settings page. Before the click was sent from Nextion to ESPHome where it was processed (including the time counting) and then a change page command was sent back from ESPHome to Nextion. I've removed this from ESPHome and put all the logic into Nextion, reducing the calls back and forward.
You probably have tested before with the ESPHome after the code was removed, but with a TFT from before the new code was moved there.
